packaging/android/build.sh

```diff
@@ -93,7 +93,7 @@ if [ -n "${QT5_ANDROID+X}" ] ; then
 	echo "Using Qt5 in $QT5_ANDROID"
 elif [ -d "$SUBSURFACE_SOURCE/../Qt/${LATEST_QT}" ] ; then
 	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/${LATEST_QT}
-elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.9.3" ] ; then
+elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.13.1" ] ; then
 	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.9.3
 elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.9.1" ] ; then
```

Comment:
NAH, that doesn't look right. If we find 5.13.1 we set it to 5.9.3? I don't think so.

Reply:
Hehe. Fumble when splitting a huge messy diff into sensible patches. Obviously missed to remove that. Now its gone.